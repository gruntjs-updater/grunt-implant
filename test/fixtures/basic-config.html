<!DOCTYPE html>
<html>
<head>
</head>
<body>
<!--implant:javascript-->
</body>
</html>