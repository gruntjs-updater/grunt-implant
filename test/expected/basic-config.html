<!DOCTYPE html>
<html>
<head>
</head>
<body>
<script src="http://cdnjs.cloudflare.com/ajax/libs/angular.js/1.3.15/angular.min.js" type="text/javascript"></script><script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-animate.js" type="text/javascript"></script><script src="http://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.13/angular-ui-router.min.js" type="text/javascript"></script>
</body>
</html>